Natives/inkText.reds

```diff
@@ -88,6 +92,60 @@ public func SetLineHeight(lineHeight: Float) -> Void {
 	this.lineHeightPercentage = lineHeight;
 }
 
+@addMethod(inkText)
+public func SetSimpleWrapPosition(wrapPosition: Float) -> Void {
+	this.simpleWrapPosition = wrapPosition;
+}
+
+@addMethod(inkText)
+public func GetSimpleWrapPosition() -> Float {
+	return this.simpleWrapPosition;
+}
+
+@addMethod(inkText)
+public func SetTextAndWrap(text: String) -> String {
+	let SPLIT_CHARS = [" ", "-", "\t"];
+
+	// Very rough calculation of the letter width (not actual pixel length, but generally close enough)
+	let letterHeight = Cast<Float>(this.GetFontSize());
+	let letterWidth = Cast<Float>(this.GetFontSize()) / 2.0;
```

Comment:
Couldn't think of a great way to get the actual pixel size - so this basically just generates a best guess or usable number. I'm sure it gets less accurate with larger text sizes. Potentially could try and use `SetText()` then get the bounds of the widget - but I didn't spend any time trying to figure that out.